.github/workflows/ci.yaml

```diff
@@ -3,17 +3,63 @@ name: default
   push:
     branches:
       - main
+      - release-*
     tags:
       - v*
-  pull_request: {}
-env:
-  CI_ARGS: --cache-from=type=registry,ref=registry.dev.siderolabs.io/${GITHUB_REPOSITORY}:buildcache --cache-to=type=registry,ref=registry.dev.siderolabs.io/${GITHUB_REPOSITORY}:buildcache,mode=max
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
```

Comment:
does it mean it will run the build each time `labeled` is touched?

Reply:
only if label is removed and added

Reply:
yes, but technically on any label change? it will re-run all steps?

Reply:
yeh, kind of, but the step will be skipped if label is not matched

Reply:
but the build will still run though... I wonder if it should be a separate workflow?

Reply:
yeh, even if separate workflow, it starts a job anyways, so not much different. Well i could have a different one that just triggers on label and skip at job level. I guess it's probably easier to read